README.md

```diff
@@ -256,6 +256,48 @@ It is not recommended to serve static assets from a Rails server in production.
 
 1. Change the value of the `use_domain` column to `true` (or `1`) in the `communities` table.
 
+
+#### Setting up S3
+
+If you want to use s3 to host your images, you need to do a bit more configuration.
+
+1. Create a IAM role which has full s3 access.  Save the aws access and secret keys.
```

Comment:
Could you make all references to S3 and AWS uppercase? That's according to Amazon's branding :)